app/controllers/brexit_checker_controller.rb

```diff
@@ -11,9 +13,17 @@ class BrexitCheckerController < ApplicationController
     expires_in(30.minutes, public: true) unless Rails.env.development?
   end
 
-  before_action :check_accounts_enabled, only: [:save_results]
+  before_action :check_accounts_enabled, only: %i[save_results saved_results]
 
   def show
+    @account_information = if logged_in? && accounts_enabled?
+                             "Logged in. <a class=\"govuk-link\" href=\"#{transition_checker_end_session_path}\">Log out.</a><br><a class=\"govuk-link\" href=\"#{transition_checker_saved_results_path}\">See previous results.</a>"
+                           elsif accounts_enabled?
+                             "Not logged in. <a class=\"govuk-link\" href=\"#{transition_checker_new_session_path}\">Login.</a>"
+                           else
+                             ""
+                           end
+
```

Comment:
Rather than something on the question pages for this, we'll want to change the call-to-action on the results page... I'm not sure if we've agreed what that should look like yet, so maybe for now just say "You are already logged in" above the green button?

Reply:
Done.  Kept the plain text for now, which can get updated to match the designs as part of a frontend card.